internal/affine/colorm.go

```diff
@@ -167,6 +167,229 @@ func (c *ColorM) UnsafeElements() ([]float32, []float32) {
 	return eb, et
 }
 
+func (c *ColorM) det() float32 {
+	if c == nil {
+		return 0
```

Comment:
When c is nil, c is an identity. Then this should return 1. (This should be included in `isInited`).